src/sentry/issues/auto_source_code_config/derived_code_mappings_endpoint.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from typing import Literal
+
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.serializers import serialize
+from sentry.integrations.source_code_management.repo_trees import (
+    RepoAndBranch,
+    RepoTreesIntegration,
+)
+from sentry.issues.auto_source_code_config.integration_utils import (
+    InstallationCannotGetTreesError,
+    InstallationNotFoundError,
+)
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+
+from .code_mapping import (
+    CodeMapping,
+    CodeMappingTreesHelper,
+    FrameInfo,
+    NeedsExtension,
+    create_code_mapping,
+)
+from .integration_utils import get_installation
+
+logger = logging.getLogger(__name__)
+
+
+def process_get_request(request: Request, organization: Organization) -> Response:
```

Comment:
Roughly equivalent to the original method:
https://github.com/getsentry/sentry/blob/88e21e7964b401f595aaa4d84ec83a6cda429ee7/src/sentry/issues/endpoints/organization_derive_code_mappings.py#L55-L78